flask/sessions.py

```diff
@@ -332,6 +332,7 @@ def open_session(self, app, request):
 
     def save_session(self, app, session, response):
         domain = self.get_cookie_domain(app)
+        is_IP(domain, "SESSION_COOKIE_DOMAIN")
```

Comment:
ideally you would use `if is_ip(domain): warnings.warn(...)` instead of having the function print something